source/templates.tex

```diff
@@ -8417,6 +8441,21 @@
 \end{codeblock}
 \end{example}
 
+\pnum
+The type of \tcode{B} in the \grammarterm{noexcept-specifier}
+\tcode{noexcept(B)} of a function type is \tcode{bool}.
```

Comment:
Do any of these need to use `\keyword`? I don't remember its use cases.

Reply:
I don't think so.  @jensmaurer @tkoeppe, thoughts?

Reply:
Yes, we should say `\tcode{\keyword{noexcept}(B)}` here I think. I can make that change, thanks for flagging!